bundles/core/src/test/java/com/adobe/cq/wcm/core/components/internal/models/v1/DownloadImplTest.java

```diff
@@ -174,19 +174,6 @@ public void testDownloadWithCustomActionText() throws Exception
         Utils.testJSONExport(download, Utils.getTestExporterJSONPath(TEST_BASE, DOWNLOAD_FULLY_CONFIGURED));
     }
 
-    @Test
-    public void testDownloadWithActionTextFromStyle() throws Exception
-    {
-        Resource mockResource = mock(Resource.class);
-        Style mockStyle = new MockStyle(mockResource, new MockValueMap(mockResource, new HashMap() {{
-            put(Download.PN_ACTION_TEXT, STYLE_ACTION_TEST);
-        }}));
-
-        Download download = getDownloadUnderTest(DOWNLOAD_2, mockStyle);
-        assertEquals("Expected action text is not correct", STYLE_ACTION_TEST, download.getActionText());
-        Utils.testJSONExport(download, Utils.getTestExporterJSONPath(TEST_BASE, DOWNLOAD_WITH_ACTION_TEXT_FROM_STYLE));
```

Comment:
Wouldn't it make sense to also remove the `DOWNLOAD_WITH_ACTION_TEXT_FROM_STYLE` constant and referenced JSON.

Reply:
Good spot, should have checked that, updated in 551b9b9